lighthouse-core/config/index.js

```diff
@@ -241,19 +305,53 @@ class Config {
       configJSON = defaultConfig;
     }
 
-    this._audits = configJSON.audits ? expandAudits(
-        filterAudits(configJSON.audits, auditWhitelist)
+    this._configJSON = this._initRequirePaths(configJSON);
```

Comment:
why not just store it on `this.json` if you're just going to make a getter for `this.json`? :P